q2_demux/_summarize/assets/src/box.js

```diff
@@ -0,0 +1,313 @@
+// Heavily modified variant of http://bl.ocks.org/jensgrubert/7789216
+// which is based off of https://gist.github.com/mbostock/4061502
+// which is written and marked as being under GPL-3
```

Comment:
This will be an issue, as all derivatives (including this PR) would need to be GPL-3